libs/logic-apps-shared/src/utils/src/lib/models/token.ts

```diff
@@ -1,5 +1,15 @@
-import type { TokenType } from '../../editor';
```

Comment:
Can this be exported from editor?

Reply:
I thought since this token.ts file is now in logic-apps-shared now it cant reference anything in libs/designer-ui/src/lib/editor.